README.md

```diff
@@ -1,2 +1,43 @@
-Bring Your Own Notebook (BYON) project repository.
---------------------------------------------------
+# Bring Your Own Notebook (BYON) Working Group
+
+The goal of the Bring Your Own Notebook (BYON) working group is to create a service that implements the BYON MVP in the context of Open Data Hub (ODH).
+
+## Phase 1
+
+The Phase 1 (MVP) of BYON aims to implement the following requirements:
+
+### Requirements
+
+Functionality for an Admin to import existing images using a URL linked to a public registry through a UI and spawn a Jupyter Notebook image which is available for selection for users through the Spawner like out-of-the-box images.
+
+* [Ability to import notebook images using a URL via a UI](https://github.com/open-services-group/byon/issues/1)
```

Comment:
It might be tricky to track that here one by one, what about linking the milestone directly: https://github.com/open-services-group/byon/milestone/1 ?

Reply:
I agree it might be good to link the Milestone here as there will be many issues in the future leading this phase, not just these few. 👍 

The issues that I linked here though, are a direct mapping from https://docs.google.com/document/d/1nd_WayY7RB3H7SlqMNlq1LsiS17CnG8V5D6UNsnmIYU/edit - which I guess are "requirements" rather than issues/actions needed to complete this Milestone. 
I think there wont be a one to one mapping, eg: for each requirement, there might be plenty of issues, so this one to one linking isnt probably the best.

Do you suggest that i remove the linking here and just keep the text, so stakeholder are aware at a glance what the requirements are and link the Milestone at the start of this list?